Anzee/PostBody.swift

```diff
@@ -0,0 +1,24 @@
+//
+//  PostBody.swift
+//  Mailchimp SDK
+//
+//  Created by Michael Patzer on 5/7/19.
+//  Copyright © 2019 Michael Patzer. All rights reserved.
+//
+
+import Foundation
+
+enum PostBody<T: Codable, U: PostBodyParamable> {
+    case object(T)
+    case params(U)
+}
+
+protocol PostBodyParamable {
+    var params: [String: String] { get }
+}
+
+struct ParametersDefault: PostBodyParamable {
+    var params: [String : String] = [:]
+}
+
+struct CodableDefault: Codable {}
```

Comment:
`ParametersDefault` and `CodableDefault` are empty default values for each of the `PostBody` enum associated values (which are generics, since `Codable` is a protocol). `extension APIRequest` has a default `func postBody()` that uses these, so that each and every object conforming to `APIRequest` doesn't have to add a `typealias` for `T` and `U`.